hubble/hubble.go

```diff
@@ -520,14 +557,26 @@ func (k *K8sHubble) Enable(ctx context.Context) error {
 			return err
 		}
 	}
-
 	if peerSvc := k.generatePeerService(); peerSvc != nil {
 		k.Log("🚀 Creating Peer Service...")
 		if _, err := k.client.CreateService(ctx, k.params.Namespace, peerSvc, metav1.CreateOptions{}); err != nil {
 			return err
 		}
 	}
 
+	// if enable metrics
+	enableHubbleMetrics, err := helmState.Values.PathValue("hubble.metrics.enabled")
```

Comment:
@aanm yep, you are right. I have change it. but I also submit `merge commit`. so I need to do something? Can you guide me?

Reply:
@misstick86 run `git rebase -i origin/master`, you should see all commits listed. You can squash all commits by adding `fixup` instead of `pick` for all of them except the first commit. After that you can push force into this branch.

Reply:
@aanm  thanks for your guidance,  now I have change it.